<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_titre">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>:::::::::::: METEO ::::::::::::</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ville">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selectioner une ville : </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ville">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_source">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>520</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Source: Openweather</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_valider">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_temp">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_vent">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_hum">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vent">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vent :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hum">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidit√©:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
